test/unit_tests/apps/owning-a-home/js/explore-rates/RateCheckerChartMenu-spec.js

```diff
@@ -44,7 +44,7 @@ describe( 'explore-rates/RateCheckerChartMenu', () => {
       xAxis: {
         categories: [ 'Jan', 'Feb' ]
       },
-      series: [ { data: [ 29.9, 71.5 ] } ]
+      series: [ { data: [ 29.9, 71.5 ]} ]
```

Comment:
We no longer need the space?

Reply:
Hmmm this was an autofix from the linter. Not sure why it triggered on that. I moved the data to its own line, like the categories above it, to make this moot.